semgrep-core/Makefile

```diff
@@ -7,7 +7,7 @@ all:
 clean:
 	dune clean
 test:
-	dune runtest
+	dune runtest -f
```

Comment:
Thx! I have no idea how dune works and why it caches those things ...

Reply:
I don't know either. `dune runtest` is a special feature they provide, and we don't have to use it. In other projects I've just built an executable and executed it. There are two ways:

* directly call `_build/....../test_foo.exe` as in https://github.com/returntocorp/ocaml-tree-sitter/blob/master/Makefile#L25
* or use `dune exec` to more elegantly locate the executable as in https://github.com/mjambon/dune-deps/blob/master/Makefile#L17

The latter is one of methods for running tests recommended in the dune manual (https://dune.readthedocs.io/en/stable/tests.html).